src/config.js

```diff
@@ -125,6 +126,7 @@ module.exports = {
     entity_minimum_width,
     highlighted_entity_icon,
     highlighted_entity_margin,
+    highlighted_entity_zindex,
```

Comment:
We don't directly need to export max_entity_width for now
    zindex_offset_multiplier,
    highlighted_entity_zindex,